curriculum/test/test-challenges.js

```diff
@@ -525,9 +525,9 @@ ${inspect(commentMap)}
           // TODO: the no-solution filtering is a little convoluted:
           const noSolution = new RegExp('// solution required');
 
-          const solutionsAsArrays = solutions.map(sortChallengeFiles);
+          const sortedSolutions = solutions.map(sortChallengeFiles);
```

Comment:
Please remind me why we sort these? Especially here.

Reply:
Here?  No reason at all. I hadn't noticed that this is only used in the `isEmpty` check.  I'll get rid of it.

In general? Choosing which order to display the editors, providing a consistent 'target editor' (i.e. the one that gets focus) fallback in case there's no editable region.